classes/student.rb

```diff
@@ -4,6 +4,7 @@ class Student < Person
   def initialize(age, classroom, name: 'unknown', parent_permession: true)
     super(age, name, parent_permession)
     @classroom = classroom
+    @classroom.add_student(self)
```

Comment:
- [ ] The association should have its method. Kindly remove the `@classroom.add_...` part and add the method for the association.

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```